### ISSUE TITLE
armhf

### ISSUE THREAD

**Issue by HrGaertner** [philpagel/debian-headless#3]
Hello nice project. But i do not get it to run with arm hf.

`sed "s/ARCH/armv7/" isolinux.cfg.template > tmp/isolinux/isolinux.cfg
/bin/sh: 1: cannot create tmp/isolinux/isolinux.cfg: Directory nonexistent
make: *** [Makefile:41: isolinux] Fehler 2
`

Thanks

**Comment by philpagel** [comment]
Hi,

I have never tried this on ARM and I'm afraid it is quite possible that it does not work on ARM at all. To my knowledge, ARM machines, like the raspberry pi, use a very different boot process compared to intel/AMD machines. Isolinux//syslinux is a bootloader for the intel/amd world and I am not sure what the ARM equivalent would be.

That said, can you give more details? E.g. what image are you using what is the target machine, what is your desired installation media and target disk? 

Maybe there is a way to make this work in a different way as I think recent versions of the PI support booting off media other than SD cards. However, I have never tried that and would have to check if I have a recent PI lying around.

**Comment by philpagel** [comment]
Just adding some links:

https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/msd.md
https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/README.md

These will not fixe the issue. I am keeping these here for reference for when I find time to look into this, at some point.

**Comment by HrGaertner** [comment]
Sorry i found an other solution. But if you want i can still help if you tell me how

**Comment by philpagel** [comment]
Hi,

Thanks for the offer. It would indeed help if you could tell me what
ARM board you are targetting (RPI? other?) and what exact debian image
you are using. Also: what is the alternative solution you are refering
to? Maybe there is something to be learned from that.

cu
    Phil

On Sat, Apr 25, 2020 at 05:56:14PM +0000, HrGaertner wrote:
> Sorry i found an other solution. But if you want i can still help if you tell
> me how
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.*
> 

-- 
Phil
Tech tinkerer
www.techbotch.org

**Comment by HrGaertner** [comment]
I used a wd my cloud mirror gen2. I flashed following to this [instructions](https://community.wd.com/t/alternative-firmware-debian-jessie-synology-dsm6/156551/606) the cloud with synology dsm but it did not work correctly and i could not access the device and reset it (web interface crashed ssh not enabled). I hoped I could boot from a USB Stick but it does not seem like that. The solution was to enter new hdds. So I did not need Debian headless and now i have ubuntu on my cloud which just runs fine. And that is why I do not need the image anymore. The Ubuntu image was installed on an other arm hf device and after this it was stuffed into a .tar.gz so all the others could install it on their cloud (also see the instructions in the link above).

Thanks for your interest

P.S. Your Website is pretty cool
